.github/workflows/haskell.yml

```diff
@@ -13,32 +13,52 @@ jobs:
 
     name: GHC ${{ matrix.ghc }} on ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v3
-    - uses: haskell/actions/setup@v2
```

Comment:
> `haskell/actions`

OK yes that repo is deprecated, I just learned. This helps me understand why these changes.

Reply:
My main goal was compiling with `javascript-backend`. I think that #56 do it so I am OK to close...